README.md

```diff
@@ -1,113 +1,28 @@
 
   [1]: https://routific.com/developers
 
+Please refer to the full [documentation](https://docs.routific.com) for a detailled documentation of the API.
+
 Installing
 ----------
 
 `gem install routific`
 
 Usage
 -----
-Remember to require it and instantiate it with your token before using it
-
-```ruby
-require 'routific'
-routific = Routific.new(--API_KEY--)
-```
-
-### Instance methods
-
-`routific.setVisit( id, [params] )`
-
-Sets a visit for the specified location using the specified parameters
-
-Required arguments in params:
-
-- location: Object representing the location of the visit.
-  + lat: Latitude of this location
-  + lng: Longitude of this location
-  + name: (optional) Name of the location
-
-Optional arguments in params:
-
- - start: the earliest time for this visit. Default value is 00:00, if not specified.
- - end: the latest time for this visit. Default value is    23:59, if not specified.
- - duration: the length of this visit in minutes
- - demand: the capacity that this visit requires
-
-`routific.setVehicle( id, params )`
-
-Sets a vehicle with the specified ID and parameters
-
-Required arguments in params:
-
-- start_location: Object representing the start location for this vehicle.
-  + lat: Latitude of this location
-  + lng: Longitude of this location
-  + name: (optional) Name of the location
-
-Optional arguments in params:
-
- - end_location: Object representing the end location for this vehicle.
-  + lat: Latitude of this location
-  + lng: Longitude of this location
-  + name: (optional) Name of the location
-
- - shift_start: this vehicle's start shift time (e.g. '08:00'). Default value is 00:00, if not specified.
- - shift_end: this vehicle's end shift time (e.g. '17:00'). Default value is 23:59, if not specified.
- - capacity: the capacity that this vehicle can load
-
-`routific.setOptions( params )`
-
-Sets optional options onto the route requests.
-
-Optional arguments must be one of the following:
-
-- traffic
-- min_visits_per_vehicle
-- balance
-- min_vehicles
-- shortest_distance
-
-`routific.getRoute()`
-
-Returns the route using the previously provided network, visits and fleet information
-
-
-### Class methods
-
-`Routific.setToken( token )`
-
-Sets the default access token to use
-
-`Routific.getRoute( id, [params] )`
-
-Returns the route using the specified access token, network, visits and fleet information
-
-
-Both getRoute functions return the Route object, which has the following attributes:
-
- - status: A sanity check, will always be success when the HTTP code is 200
- - fitness: Total travel-time, representing the fitness score of the solution (less is better)
- - unserved: List of visits that could not be scheduled.
- - vehicleRoutes: The optimized schedule
-
-Examples
---------
-Example 1:
+Remember to require it and set your token before using it
 
 ```ruby
 require 'routific'
+routific = Routific.set_token(--API_KEY--)
```

Comment:
`set_token` doesn't return a Ruby instance; should it?

Reply:
Currently we need to do:
```ruby
require 'routific'
Routific.set_token(--API_KEY--)
routific = Routific.new
...
```

Reply:
You are right